test_gtest/ParenthesesTests.cpp

```diff
@@ -14,3 +14,55 @@ TEST(ParenthesesTests, givenEmptyStringWhenIsBalancedIsCalledThenResultIsTrue)
     // THEN
     ASSERT_TRUE(result);
 }
+
+TEST(ParenthesesTests, givenCorrectTwoParentheses)
+{
+    // GIVEN
+    std::string input{"()"};
+    Parentheses p;
```

Comment:
W każdym teście macie Parentheses p;
Zróbcie klasę ParenthesesTests, w której to umieścicie, a TEST zamieńcie na TEST_F. Dzięki temu oszczędzicie po 1 linijce w każdym teście.

Reply:
done